edgar-app/src/components/dashboardPages/documents/DocumentFirstFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { HStack, Menu, MenuButton, MenuList, MenuItem, Icon } from '@chakra-ui/react';
+
+import UpArrowIcon from 'assets/icons/Arrow/UpArrowIcon';
+import DownArrowIcon from 'assets/icons/Arrow/DownArrowIcon';
+import CalendarIcon from 'assets/icons/CalendarIcon';
+
+interface DocumentFirstFilterProps {
+  onFilterChange: (value: string) => void;
+  sortOrder: string;
+}
+
+const DocumentFirstFilter: React.FC<DocumentFirstFilterProps> = ({ onFilterChange, sortOrder }) => {
+  const getIcon = (menuItemValue: string) => {
+    switch (menuItemValue) {
+      case 'asc':
+        return <Icon as={UpArrowIcon} w="12px" h="12px" />;
+      case 'desc':
+        return <Icon as={DownArrowIcon} w="12px" h="12px" />;
+      case 'newest':
+      case 'oldest':
+        return <Icon as={CalendarIcon} />;
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <HStack spacing="8px">
+      <Menu defaultIsOpen>
+        <MenuButton as={HStack} bg="white" p="4px 12px" border="1px" borderColor="blue.700" borderRadius="32">
+          <span>Options de tri</span>

```

Comment:
When using Chakra-UI you never need to use HTML tag (unless otherwise specificized, like with `<Select>` and `<option>`).
If you want to have a `<span>`, you need to use: `<Box as="span" />`

There is no such title inside the filter menu => so no need to add it